src/app/services/car.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { API_URL } from '../config/config';
```

Comment:
Great that you've externalized this configuration, but instead of creating your own config file, use angular environment file : https://angular.io/guide/build#configuring-application-environments